views/deductions/political.pug

```diff
@@ -0,0 +1,24 @@
+extends ../base
+include ../_includes/yesNoRadios
+
+block variables
+  -var title = __('Political contributions')
+  -var politicalClaim = hasData(data, 'deductions.politicalClaim') ? data.deductions.politicalClaim : ''
+
+block content
+
+  h1 #{title}
+
+  div
+    p #{__('Did you make any political contributions during the current tax year (filing for 2018)?')}
+
+  form.cra-form(method='post')
+    +yesNoRadios('politicalClaim', politicalClaim, 'Deduct political contributions?', errors)
+
+    input#redirect(name='redirect', type='hidden', value='/deductions/donations')
+
+    .buttons
```

Comment:
Again, we can use the button mixin here too.

Reply:
done